<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adherentsdialog</class>
 <widget class="QDialog" name="adherentsdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color : #181DA4 ;
border-radius : 20px ;
color : white ;
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset>
      <normaloff>71.png</normaloff>71.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Adherents Liste</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_delete">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/45.png</normaloff>:/new/prefix1/45.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_init">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>390</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/46.png</normaloff>:/new/prefix1/46.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Code Adhérent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_code_adhr">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nom Adhérent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nom_adr"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nombre d'emprunts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nbrEmp_adr"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/33.png</normaloff>:/new/prefix1/33.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_update">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>310</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/59.png</normaloff>:/new/prefix1/59.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_status2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>381</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_load">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>480</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/42.png</normaloff>:/new/prefix1/42.png</iconset>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>Adhérent Info</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>571</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>Adhérent Table</string>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>531</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <zorder>groupBox</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>pushButton_load</zorder>
    <zorder>pushButton_delete</zorder>
    <zorder>pushButton_init</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>pushButton_save</zorder>
    <zorder>pushButton_update</zorder>
    <zorder>label_status2</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/prefix1/55.png</normaloff>:/new/prefix1/55.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Stats</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_stats_adh">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>470</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background-color : #181DA4 ;
border-radius : 20px ;
color : white ;
}</string>
     </property>
     <property name="text">
      <string>SHOW</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/55.png</normaloff>:/new/prefix1/55.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>891</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
